takeshape/penny
# Pagination of product collection pages

### Test Plan (Steps to test):

1. Check out preview site, go to `/collection/men`
1. Browse through the 3 pages, any issues?
1. Full reload on the 2nd or 3rd page. Can you page back to 1?
2. Review the homepage for any regressions in the product module
3. In the shared TakeShape project, add a slug for a collection
4. Go to the new URL `/collection/YOUR_SLUG`. It should load, albeit more slowly (it hasn't been rendered yet).
